eatngo-core/src/main/kotlin/com/eatngo/store/service/StoreTotalStockService.kt

```
@@ -0,0 +1,149 @@
+        // 위에서 로컬 캐시로 해결 안 되면 DB 조회
+        return try {
+            if (shouldAllowDbQuery(storeId)) {
+                val dbStock = calculateTotalStockFromDBWithLimit(storeId)
+                localCache[cacheKey] = CachedStockInfo(dbStock)
+                
+                dbStock
+            } else {
+                // DB 조회도 제한된 경우 - 기본값 반환
+                getDefaultStock(storeId)
+            }
+        } catch (dbEx: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)